src/models/connection/from_client/connection_from_client.ts

```diff
@@ -140,18 +190,42 @@ export class ConnectionFromClient {
 
   // eslint-disable-next-line
   private async validateJira(tool: any, errorPrefix: string, errors: string[]): Promise<boolean> {
-    // TODO Implement
+    ToolType.JIRA.attributesFromClient.forEach((attribute) => {
+      if (!tool[attribute] == undefined) {
+        errors.push(`${errorPrefix} is missing ${attribute}`);
+        return false;
+      }
+    })
+    const jiraApiKey: string = tool.jiraApiKey
+    const jiraUserEmail: string = tool.jiraUserEmail
+    const jiraDomain: string = tool.jiraDomain
+
+    const jiraConnection: superagent.Response | number = await getJiraIssueStatuses(jiraDomain, jiraApiKey, jiraUserEmail)
+    if (!jiraConnection || typeof jiraConnection === 'number') {
+      errors.push(`Invalid Jira ApiKey, domain or user email`)
```

Comment:
Zde ta hláška by měla být nějaká lepší. Pokud nemáme lepší způsob jak ověřit credentials.

Reply:
teď nebudeme měnit